templates/.circleci/config.yml

```diff
@@ -4,8 +4,12 @@ unit_tests: &unit_tests
     - restore_cache:
         key: dependency-cache-{{ checksum "package-lock.json" }}
     - run:
-        name: NPM Rebuild
-        command: npm install
+        name: Install Latest NPM.
+        # npm@3 is buggy
+        command: if [[ $(npm -v | cut -c -1) > 3 ]] ; then npm i -g npm@latest; else echo "Skip npm updating"; fi
+    - run:
+        name: NPM Install.
+        command: npm ci || npm i
     - run:
         name: Run unit tests.
```

Comment:
```
name: Test
```

Reply:
@michael-ciniawsky All name should be have action, it is better read in CircleCI interface. Use `Run test`.